dist/react-experiments.js

```diff
@@ -171,65 +151,75 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	"use strict";
 
-	var DEFAULT_EXPERIMENT_COMPONENT = __webpack_require__(4).DEFAULT_EXPERIMENT_COMPONENT;
+	var _interopRequire = function (obj) { return obj && obj.__esModule ? obj["default"] : obj; };
 
-	module.exports = {
+	var React = _interopRequire(__webpack_require__(2));
 
-	  enrollInExperiment: function enrollInExperiment(component, child, variationName) {
-	    if (variationName && child.props.name === variationName) {
-	      component.exposedVariation = child;
-	    } else if (child.props.displayName === DEFAULT_EXPERIMENT_COMPONENT) {
-	      component.defaultComponent = child;
-	    }
-	    return component;
+	var Parametrize = React.createClass({
+	  displayName: "Parametrize",
+
+	  getInitialState: function getInitialState() {
+	    return {
+	      experimentParameters: null
+	    };
 	  },
 
-	  getExposedExperimentVariation: function getExposedExperimentVariation(childrenComponents, variationName) {
-	    var _this = this;
+	  childContextTypes: {
+	    experimentParameters: React.PropTypes.object.isRequired,
+	    experimentProps: React.PropTypes.object.isRequired
+	  },
 
-	    if (!childrenComponents.reduce) {
-	      return this.enrollInExperiment({}, childrenComponents, variationName);
-	    }
+	  getChildContext: function getChildContext() {
+	    return {
+	      experimentParameters: this.state.parameters,
+	      experimentProps: this.props
+	    };
+	  },
 
-	    return childrenComponents.reduce(function (component, child) {
-	      if (component.exposedVariation) {
-	        return component;
-	      }
+	  componentDidMount: function componentDidMount() {
+	    this.selectVariation();
+	  },
 
-	      return _this.enrollInExperiment(component, child, variationName);
-	    }, {});
-	  }
-	};
+	  selectVariation: function selectVariation() {
+	    var experiment = this.props.experimentClass;
 
-/***/ },
-/* 4 */
-/***/ function(module, exports) {
+	    if (!experiment || !experiment.inputs) {
+	      console.error("You must pass in an experimentClass instance as a prop");
+	      return;
+	    }
 
-	"use strict";
+	    this.setState({
+	      parameters: experiment.getParams()
+	    });
+	  },
 
-	Object.defineProperty(exports, "__esModule", {
-	  value: true
-	});
-	var DEFAULT_EXPERIMENT_COMPONENT = "DEFAULT";
-	exports.DEFAULT_EXPERIMENT_COMPONENT = DEFAULT_EXPERIMENT_COMPONENT;
+	  renderExperiment: function renderExperiment() {
+	    var _this = this;
 
-/***/ },
-/* 5 */
-/***/ function(module, exports) {
+	    if (!this.state.parameters) {
+	      return null;
+	    }
 
-	"use strict";
+	    var renderedChildren = React.Children.map(this.props.children, function (child) {
+	      return React.addons.cloneWithProps(child, { experimentParameters: _this.state.parameters, experimentProps: _this.props });
```

Comment:
in theory I should be able to use https://facebook.github.io/react/docs/top-level-api.html#react.cloneelement, but I think I must need to update the version of the library in the package.json or something since it was not working locally with this :(